UnityProject/Assets/Scripts/Items/PDA/PDALogic.cs

```diff
@@ -458,6 +458,9 @@ public void SpawnUplinkItem(GameObject objectRequested, int cost)
 			{
 				UplinkTC -= cost;
 				var item = result.GameObject;
+				if(item.TryGetComponent<RandomItemSpot>(out var randomItem)){
```

Comment:
it would be better before you spawned the random object spawn, then you spawn it in, 
you pick the object before Spawn it in I mean

Reply:
It might just be because I'm tired, but I have no idea what this means.

Reply:
Requested changes made. Marking this as resolved.